### ISSUE TITLE
训练自己模型时，开启GPU，会退出。CPU能正常运行，但比较慢。

### ISSUE THREAD

**Issue by chenhao86** [PaddlePaddle/PaddleOCR#10915]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Windows 10
- 版本号/Version：Paddle：gpu - 2.5  PaddleOCR： 2.7 问题相关组件/Related components： cuda 11.8
- 运行指令/Command Code： train.py --config=configs/rec/PP-OCRv3/en_PP-OCRv3_rec.yml 
- 完整报错/Complete Error Message：

**CUDA检测正常。但模型训练时，启用GPU会异常退出。**
### CUDA检测日志：
PyDev console: starting.
Python 3.11.4 | packaged by Anaconda, Inc. | (main, Jul  5 2023, 13:47:18) [MSC v.1916 64 bit (AMD64)] on win32
import paddle
import paddle
paddle.utils.run_check()
Running verify PaddlePaddle program ... 
I0915 13:53:11.358312  5540 interpretercore.cc:237] New Executor is Running.
W0915 13:53:11.359309  5540 gpu_resources.cc:96] The GPU architecture in your current machine is Pascal, which is not compatible with Paddle installation with arch: 70 75 80 86 , it is recommended to install the corresponding wheel package according to the installation information on the official Paddle website.
W0915 13:53:11.376264  5540 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 11.8, Runtime API Version: 11.8
W0915 13:53:11.386237  5540 gpu_resources.cc:149] device: 0, cuDNN Version: 8.9.
I0915 13:53:14.920311  5540 interpreter_util.cc:518] Standalone Executor is Used.
PaddlePaddle works well on 1 GPU.
PaddlePaddle is installed successfully! Let's start deep learning with PaddlePaddle now.

### 控制台日志：
[2023/09/15 16:14:56] ppocr INFO: Architecture : 
[2023/09/15 16:14:56] ppocr INFO:     Backbone : 
[2023/09/15 16:14:56] ppocr INFO:         last_conv_stride : [1, 2]
[2023/09/15 16:14:56] ppocr INFO:         last_pool_kernel_size : [2, 2]
[2023/09/15 16:14:56] ppocr INFO:         last_pool_type : avg
[2023/09/15 16:14:56] ppocr INFO:         name : MobileNetV1Enhance
[2023/09/15 16:14:56] ppocr INFO:         scale : 0.5
[2023/09/15 16:14:56] ppocr INFO:     Head : 
[2023/09/15 16:14:56] ppocr INFO:         head_list : 
[2023/09/15 16:14:56] ppocr INFO:             CTCHead : 
[2023/09/15 16:14:56] ppocr INFO:                 Head : 
[2023/09/15 16:14:56] ppocr INFO:                     fc_decay : 1e-05
[2023/09/15 16:14:56] ppocr INFO:                 Neck : 
[2023/09/15 16:14:56] ppocr INFO:                     depth : 2
[2023/09/15 16:14:56] ppocr INFO:                     dims : 64
[2023/09/15 16:14:56] ppocr INFO:                     hidden_dims : 120
[2023/09/15 16:14:56] ppocr INFO:                     name : svtr
[2023/09/15 16:14:56] ppocr INFO:                     use_guide : True
[2023/09/15 16:14:56] ppocr INFO:             SARHead : 
[2023/09/15 16:14:56] ppocr INFO:                 enc_dim : 512
[2023/09/15 16:14:56] ppocr INFO:                 max_text_length : 25
[2023/09/15 16:14:56] ppocr INFO:         name : MultiHead
[2023/09/15 16:14:56] ppocr INFO:     Transform : None
[2023/09/15 16:14:56] ppocr INFO:     algorithm : SVTR_LCNet
[2023/09/15 16:14:56] ppocr INFO:     model_type : rec
[2023/09/15 16:14:56] ppocr INFO: Eval : 
[2023/09/15 16:14:56] ppocr INFO:     dataset : 
[2023/09/15 16:14:56] ppocr INFO:         data_dir : ./train_data
[2023/09/15 16:14:56] ppocr INFO:         label_file_list : ['train_data/rec/rec_gt_test.txt']
[2023/09/15 16:14:56] ppocr INFO:         name : SimpleDataSet
[2023/09/15 16:14:56] ppocr INFO:         transforms : 
[2023/09/15 16:14:56] ppocr INFO:             DecodeImage : 
[2023/09/15 16:14:56] ppocr INFO:                 channel_first : False
[2023/09/15 16:14:56] ppocr INFO:                 img_mode : BGR
[2023/09/15 16:14:56] ppocr INFO:             MultiLabelEncode : None
[2023/09/15 16:14:56] ppocr INFO:             RecResizeImg : 
[2023/09/15 16:14:56] ppocr INFO:                 image_shape : [3, 48, 320]
[2023/09/15 16:14:56] ppocr INFO:             KeepKeys : 
[2023/09/15 16:14:56] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2023/09/15 16:14:56] ppocr INFO:     loader : 
[2023/09/15 16:14:56] ppocr INFO:         batch_size_per_card : 16
[2023/09/15 16:14:56] ppocr INFO:         drop_last : False
[2023/09/15 16:14:56] ppocr INFO:         num_workers : 4
[2023/09/15 16:14:56] ppocr INFO:         shuffle : False
[2023/09/15 16:14:56] ppocr INFO: Global : 
[2023/09/15 16:14:56] ppocr INFO:     cal_metric_during_train : False
[2023/09/15 16:14:56] ppocr INFO:     character_dict_path : ppocr/utils/en_dict_2.txt
[2023/09/15 16:14:56] ppocr INFO:     checkpoints : None
[2023/09/15 16:14:56] ppocr INFO:     debug : False
[2023/09/15 16:14:56] ppocr INFO:     distributed : False
[2023/09/15 16:14:56] ppocr INFO:     epoch_num : 100
[2023/09/15 16:14:56] ppocr INFO:     eval_batch_step : [0, 500]
[2023/09/15 16:14:56] ppocr INFO:     infer_img : doc/imgs_words/ch/word_1.jpg
[2023/09/15 16:14:56] ppocr INFO:     infer_mode : False
[2023/09/15 16:14:56] ppocr INFO:     log_smooth_window : 20
[2023/09/15 16:14:56] ppocr INFO:     max_text_length : 25
[2023/09/15 16:14:56] ppocr INFO:     pretrained_model : ./pretrain_models/en_PP-OCRv3_rec_train/best_accuracy
[2023/09/15 16:14:56] ppocr INFO:     print_batch_step : 10
[2023/09/15 16:14:56] ppocr INFO:     save_epoch_step : 10
[2023/09/15 16:14:56] ppocr INFO:     save_inference_dir : None
[2023/09/15 16:14:56] ppocr INFO:     save_model_dir : ./output/v3_en_mobile
[2023/09/15 16:14:56] ppocr INFO:     save_res_path : ./output/rec/predicts_ppocrv3_en.txt
[2023/09/15 16:14:56] ppocr INFO:     use_gpu : True
[2023/09/15 16:14:56] ppocr INFO:     use_space_char : False
[2023/09/15 16:14:56] ppocr INFO:     use_visualdl : False
[2023/09/15 16:14:56] ppocr INFO: Loss : 
[2023/09/15 16:14:56] ppocr INFO:     loss_config_list : 
[2023/09/15 16:14:56] ppocr INFO:         CTCLoss : None
[2023/09/15 16:14:56] ppocr INFO:         SARLoss : None
[2023/09/15 16:14:56] ppocr INFO:     name : MultiLoss
[2023/09/15 16:14:56] ppocr INFO: Metric : 
[2023/09/15 16:14:56] ppocr INFO:     ignore_space : False
[2023/09/15 16:14:56] ppocr INFO:     main_indicator : acc
[2023/09/15 16:14:56] ppocr INFO:     name : RecMetric
[2023/09/15 16:14:56] ppocr INFO: Optimizer : 
[2023/09/15 16:14:56] ppocr INFO:     beta1 : 0.9
[2023/09/15 16:14:56] ppocr INFO:     beta2 : 0.999
[2023/09/15 16:14:56] ppocr INFO:     lr : 
[2023/09/15 16:14:56] ppocr INFO:         learning_rate : 0.001
[2023/09/15 16:14:56] ppocr INFO:         name : Cosine
[2023/09/15 16:14:56] ppocr INFO:         warmup_epoch : 5
[2023/09/15 16:14:56] ppocr INFO:     name : Adam
[2023/09/15 16:14:56] ppocr INFO:     regularizer : 
[2023/09/15 16:14:56] ppocr INFO:         factor : 3e-05
[2023/09/15 16:14:56] ppocr INFO:         name : L2
[2023/09/15 16:14:56] ppocr INFO: PostProcess : 
[2023/09/15 16:14:56] ppocr INFO:     name : CTCLabelDecode
[2023/09/15 16:14:56] ppocr INFO: Train : 
[2023/09/15 16:14:56] ppocr INFO:     dataset : 
[2023/09/15 16:14:56] ppocr INFO:         data_dir : ./train_data
[2023/09/15 16:14:56] ppocr INFO:         ext_op_transform_idx : 1
[2023/09/15 16:14:56] ppocr INFO:         label_file_list : ['train_data/rec/rec_gt_train.txt']
[2023/09/15 16:14:56] ppocr INFO:         name : SimpleDataSet
[2023/09/15 16:14:56] ppocr INFO:         transforms : 
[2023/09/15 16:14:56] ppocr INFO:             DecodeImage : 
[2023/09/15 16:14:56] ppocr INFO:                 channel_first : False
[2023/09/15 16:14:56] ppocr INFO:                 img_mode : BGR
[2023/09/15 16:14:56] ppocr INFO:             RecConAug : 
[2023/09/15 16:14:56] ppocr INFO:                 ext_data_num : 2
[2023/09/15 16:14:56] ppocr INFO:                 image_shape : [48, 320, 3]
[2023/09/15 16:14:56] ppocr INFO:                 max_text_length : 25
[2023/09/15 16:14:56] ppocr INFO:                 prob : 0.5
[2023/09/15 16:14:56] ppocr INFO:             RecAug : None
[2023/09/15 16:14:56] ppocr INFO:             MultiLabelEncode : None
[2023/09/15 16:14:56] ppocr INFO:             RecResizeImg : 
[2023/09/15 16:14:56] ppocr INFO:                 image_shape : [3, 48, 320]
[2023/09/15 16:14:56] ppocr INFO:             KeepKeys : 
[2023/09/15 16:14:56] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2023/09/15 16:14:56] ppocr INFO:     loader : 
[2023/09/15 16:14:56] ppocr INFO:         batch_size_per_card : 16
[2023/09/15 16:14:56] ppocr INFO:         drop_last : True
[2023/09/15 16:14:56] ppocr INFO:         num_workers : 0
[2023/09/15 16:14:56] ppocr INFO:         shuffle : True
[2023/09/15 16:14:56] ppocr INFO: profiler_options : None
[2023/09/15 16:14:56] ppocr INFO: train with paddle 2.5.1 and device Place(gpu:0)
[2023/09/15 16:14:56] ppocr INFO: Initialize indexs of datasets:['train_data/rec/rec_gt_train.txt']
list index out of range
[2023/09/15 16:14:56] ppocr INFO: Initialize indexs of datasets:['train_data/rec/rec_gt_test.txt']
W0915 16:14:56.967911 15292 gpu_resources.cc:96] The GPU architecture in your current machine is Pascal, which is not compatible with Paddle installation with arch: 70 75 80 86 , it is recommended to install the corresponding wheel package according to the installation information on the official Paddle website.
W0915 16:14:56.967911 15292 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 11.8, Runtime API Version: 11.8
W0915 16:14:56.970903 15292 gpu_resources.cc:149] device: 0, cuDNN Version: 8.9.

Process finished with exit code -1073740791 (0xC0000409)

**Comment by jiangbossB** [comment]
跟你的一样，请问解决了吗

**Comment by chenhao86** [comment]
还没能解决。直接把GPU关了，用cpu跑的，就是有些慢。

**Comment by jiangbossB** [comment]
> 还没能解决。直接把GPU关了，用cpu跑的，就是有些慢。

我去，cpu猴年马月也跑不完啊

**Comment by chenhao86** [comment]
奇怪了。  我把  --config=configs/rec/rec_r34_vd_none_bilstm_ctc.yml 指向这个文件，竟然可以用GPU。但acc=0，也没有生成 best_accuracy。

**Comment by jiangbossB** [comment]
用mmocr吧   这个项目太辣鸡了  问问题都没多少人回的

**Comment by chenhao86** [comment]
@jiangbossB  mmocr  我看了下，好像只有python版本的。ppocr还行，主要是   configs 下面的  yml   ，有好多个，好像现在android  还有lite版，  fastdeploy 版，好多版本， 搞得有点迷茫了。

**Comment by linchen111** [comment]
> 用mmocr吧 这个项目太辣鸡了 问问题都没多少人回的

mmocr好像没啥支持中文的OCR模型？

**Comment by chenhao86** [comment]
直接 在AIStudio里面开GPU 跑，速度 还是可以的。
